python/graphstorm/model/utils.py

```diff
@@ -872,6 +872,33 @@ def save_full_node_embeddings(g, save_embed_path,
 
     save_shuffled_node_embeddings(shuffled_embs, save_embed_path, save_embed_format)
 
+def load_gsgnn_embeddings(emb_path, g):
```

Comment:
we didn't have a function to load GNN embeddings saved by GraphStorm? @classicsong